apps/civicsignalblog/payload.config.ts

```diff
@@ -37,6 +39,8 @@ const cors =
     ?.map((d) => d.trim())
     ?.filter(Boolean) ?? [];
 
+const customHeaders: string[] = ["x-current-app"];
```

Comment:
```suggestion
const customHeaders: string[] = ["CS-App"];
```

1. `X-` is already [deprecated](https://www.rfc-editor.org/rfc/rfc6648) so lets just use `Cs-` for CivicSignal or something along those lines
2. `current` is cool for long session i.e. backend UI, but for a header that can change on every request, I don't think it adds any value.